### PR TITLE
Add playbook to configure ceph cluster monitoring

### PR DESCRIPTION
**Pull request by dougszumski**
The logs have wildly varying formats so there still remains some
effort to parse them, but at least they are searchable.

### CODE REVIEW

File: ansible/group_vars/all/alaska [hunk 1]
```diff
@@ -18,7 +18,12 @@ project_name: p3
 
 # OpenStack credentials for posting logs and metrics for the P3 project
 monasca_agent_p3_username: p3-monasca-agent
-monasca_agent_p3_password: "{{ vault_monasca_agent_password }}"
+monasca_agent_p3_password: "{{ vault_p3_monasca_agent_password }}"
+
+# OpenStack credentials for posting logs and metrics to the central monitoring project
+monasca_project_name: monasca
```
Comment (dougszumski) [inline ansible/group_vars/all/alaska]: This needs changing to p3

File: ansible/monitoring-ceph.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  hosts: ceph
+  become: true
+  roles:
+  - role: stackhpc.monasca-rsyslog
+    monasca_rsyslog_api_auth:
+      auth_url: "http://{{ controller_vip }}:5000"
+      project: "{{ monasca_project_name }}"
+      username: "{{ monasca_agent_username }}"
+      password: "{{ monasca_agent_password }}"
+    monasca_rsyslog_venv: "/usr/libexec/monasca-rsyslog"
```
Comment (dougszumski) [inline ansible/monitoring-ceph.yml]: can be removed since it's the default

File: README.md [hunk 3]
```diff
@@ -157,3 +157,8 @@ To setup hyperconverged storage and mount storage on OpenHPC node:
 
     ansible-playbook --vault-password-file vault-password -e @config/openhpc.yml -i ansible/inventory-openhpc ansible/cluster-infra-configure.yml
 
+# Monitoring Ceph
+
+To configure monitoring of the Ceph cluster
+
+ansible-playbook --vault-password-file vault.pass -i ansible/inventory-ceph ansible/monitoring-ceph.yml
```
Comment (brtknr) [inline README.md]: Does inventory-ceph exist in the directory tree?

File: ansible/group_vars/all/secrets [hunk 4]
```diff
@@ -1,11 +1,14 @@
 $ANSIBLE_VAULT;1.1;AES256
-39363263323231633231396432363163383464643230616461333630366234663232343835643933
```
Comment (brtknr) [inline ansible/group_vars/all/secrets]: why has this changed?

File: ansible/group_vars/all/alaska [hunk 5]
```diff
@@ -21,7 +21,7 @@ monasca_agent_p3_username: p3-monasca-agent
 monasca_agent_p3_password: "{{ vault_p3_monasca_agent_password }}"
 
 # OpenStack credentials for posting logs and metrics to the central monitoring project
-monasca_project_name: monasca
+monasca_project_name: p3
```
Comment (dougszumski) [inline ansible/group_vars/all/alaska]: We still want monasca for rsyslog

File: ansible/monitoring-ceph.yml [hunk 6]
```diff
@@ -29,5 +28,6 @@
     monasca_region_name: RegionOne
     monasca_service_type: monitoring
     monasca_log_level: INFO
+    monasca_agent_system_only: true
```
Comment (dougszumski) [inline ansible/monitoring-ceph.yml]: I guess you might be looking at this, but you'll need to set `monasca_checks` on the node you want to run the ceph plugin on. Example: https://gitlab.ecdf.ed.ac.uk/stackhpc/ecdf-kayobe-config/merge_requests/5/diffs

File: ansible/group_vars/all/secrets (hunk 4 above)
Comment (Wasaac) [reply ansible/group_vars/all/secrets]: It adds vault_p3_monasca_agent_password to differentiate it from vault_monasca_agent_password.

File: ansible/group_vars/all/alaska (hunk 5 above)
Comment (Wasaac) [reply ansible/group_vars/all/alaska]: Fair point!

File: ansible/monitoring-ceph.yml (hunk 6 above)
Comment (Wasaac) [reply ansible/monitoring-ceph.yml]: Surprisingly - wasn't necessary. Seem to pick up the plugins anyway:
```
Successfully initialized checks: ['load', 'network', 'process', 'ntp', 'ceph', 'memory', 'disk', 'cpu']
```

**Pull request merged**